app/test_main.py

```diff
@@ -1,3 +1,38 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_in_human_years,dog_in_human_years,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ],
+    ids=[
+        "Age = 0",
+        "Cat & Dog age < 15",
+        "Cat & Dog age = 15",
+        "Cat & Dog age between 15 and 23",
+        "Cat & Dog age = 24",
+        "Cat & Dog age = 27",
+        "Cat & Dog age = 28",
+        "Cat & Dog age = 28"
```

Comment:
The test case identifiers for 'Cat & Dog age = 28' are duplicated. Consider changing one of them to a unique identifier to avoid confusion and ensure clarity in test results.